script.js

```diff
@@ -0,0 +1,135 @@
+const speakersObject = {
+  speakerOne: {
+    picture: 'images/face1-b.jpg',
+    name: 'John Smith',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+  speakerTwo: {
+    picture: 'images/face2-b.jpg',
+    name: 'Jane Apple',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+  speakerThree: {
+    picture: 'images/face3-b.jpg',
+    name: 'Fred Prodsis',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+  speakerFour: {
+    picture: 'images/face4-b.jpg',
+    name: 'Megane Froths',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+  speakerFive: {
+    picture: 'images/face5-b.jpg',
+    name: 'Ossan Droth',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+  speakerSix: {
+    picture: 'images/face6-b.jpg',
+    name: 'Roths Mirde',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+};
+
+const speakerArray = Object.keys(speakersObject);
```

Comment:
Not serious, but you could make `speakersObject` an array from the start 🤷

Reply:
Thanks, yeah makes more sense.